frontend/models/feedback-task-model/global-question/GlobalQuestion.model.js

```diff
@@ -0,0 +1,22 @@
+import { Model } from "@vuex-orm/core";
+
+class GlobalQuestion extends Model {
+  static entity = "globalQuestions";
+  static primaryKey = ["component_type", "question"];
+
+  static fields() {
+    return {
+      id: this.uid(),
+      dataset_id: this.attr(null),
+      order: this.number(0),
+      question: this.string(""),
+      outputs: this.attr([]),
```

Comment:
What's the purpose of this field?

Reply:
The purpose of outputs It's to initiate the kind of answer(s) used by default in the components of the question form.
Each component (singleLabel, textArea, Rating, etc) have the same shape of outputs

Reply:
As discussed, I rename all "output" by "option" to remove confusion